src/JSON For mIRC.mrc

```diff
@@ -746,7 +746,7 @@ alias JSON {
   jfm_log -I $!JSON( $+ %Args $+ ) $+ $iif($len($prop), . $+ $prop)
 
   ;; If the alias was called without any inputs
-  if (!$0) || ($0 == 1 && $1 == $null) {
+  if (!$0) || ($0 == 1) && ($1 == $null) {
```

Comment:
This is incorrect. The correct version would be the following but I chose not to include the extra `()`:
if (!$0) || (($0 == 1) && ($1 == $null)) {